vimeo/upload.py

```diff
@@ -43,6 +44,49 @@ def replace(self, video_uri, filename):
 
         return self._perform_upload(filename, ticket)
 
+    def upload_tus(self, path, filename = None):
+        """Uploads a file through the tus protocol, calls the _perform_tus_upload
+
+        Args:
+            path (string): path on disk to the file
+            filename (string, optional): name of the video file on vimeo.com
+        """
+        if filename is None:
+            filename = path
+        tus = self.post(
+            self.UPLOAD_ENDPOINT,
+            data={
+                'type': 'tus',
+                'filename': filename,
+                'size': os.stat(path).st_size
+            },
+            params={'fields': 'upload_link'}
+        )
+        return self._perform_tus_upload(tus, path, filename)
+
+    def _perform_tus_upload(self, tus_resp, path, filename):
+        """Takes a tus url and performs the actual upload.
+
+        Args:
+            tus_resp (:obj): requests object
+            path (string): path on disk to file
+            filename (string): name of the video file on vimeo.com
+        """
+        if tus_resp.status_code != 201:
+            raise UploadTicketCreationFailure(tus, "Failed to create tus upload")
+
+        tus_resp = tus_resp.json()
+        upload_link = tus_resp.get("upload_link")
+        try:
+            with io.open(filename, 'rb') as fs:
+                tus_client = client.TusClient("https://files.tus.vimeo.com")
```

Comment:
Can this be a hard coded constant? ```TUS_CLIENT_URL``` or something like that.

Reply:
@Onebrownsound It could, it's currently like that due to other limitations on regional support. I need to check in with @Dashron to make sure if we're doing region support in SDKs.